cpp/src/parquet/encoding_test.cc

```diff
@@ -641,27 +642,38 @@ class EncodingAdHocTyped : public ::testing::Test {
 
   static std::shared_ptr<::arrow::DataType> arrow_type();
 
-  void Plain(int seed) {
-    auto values = GetValues(seed);
+  void Plain(int seed, int round = 1) {
+    auto random_array = GetValues(seed);
     auto encoder = MakeTypedEncoder<ParquetType>(
         Encoding::PLAIN, /*use_dictionary=*/false, column_descr());
     auto decoder = MakeTypedDecoder<ParquetType>(Encoding::PLAIN, column_descr());
 
-    ASSERT_NO_THROW(encoder->Put(*values));
+    for (int i = 0; i < round; ++i) {
+      ASSERT_NO_THROW(encoder->Put(*random_array));
+    }
+    std::shared_ptr<::arrow::Array> values;
+    if (round == 1) {
+      values = random_array;
+    } else {
+      ::arrow::ArrayVector arrays;
+      arrays.resize(round, random_array);
```

Comment:
Small simplification:
```suggestion
      ::arrow::ArrayVector arrays(round, random_array);
```